### PR TITLE
CXX-848 Remove unused oplog_replay option from find options

### PR DESCRIPTION
**Pull request by samantharitter**

### CODE REVIEW

File: src/mongocxx/private/libmongoc_symbols.hpp [hunk 1]
```diff
@@ -126,14 +126,12 @@ MONGOCXX_LIBMONGOC_SYMBOL(uri_get_write_concern)
 MONGOCXX_LIBMONGOC_SYMBOL(uri_new)
 MONGOCXX_LIBMONGOC_SYMBOL(write_concern_copy)
 MONGOCXX_LIBMONGOC_SYMBOL(write_concern_destroy)
-MONGOCXX_LIBMONGOC_SYMBOL(write_concern_get_fsync)
```
Comment (hanumantmk) [inline src/mongocxx/private/libmongoc_symbols.hpp]: what happened here?

**Comment by Machyne** [comment]
LGTM!

File: src/mongocxx/private/libmongoc_symbols.hpp (hunk 1 above)
Comment (hanumantmk) [inline src/mongocxx/private/libmongoc_symbols.hpp]: nevermind

**Comment by hanumantmk** [comment]
lgtm

**Pull request merged**